test/files/cppy_source/test_extended_xsort.cppy

```diff
@@ -173,9 +173,9 @@ namespace xt
         auto a1 = xt::argpartition(py_a, 4, 1);
         auto a2 = xt::argpartition(py_a, 3, 2);
 
-        EXPECT_EQ(xt::view(py_a0, 2, all(), all()), xt::view(a0, 2, all(), all()));
-        EXPECT_EQ(xt::view(py_a1, all(), 4, all()), xt::view(a1, all(), 4, all()));
-        EXPECT_EQ(xt::view(py_a2, all(), all(), 3), xt::view(a2, all(), all(), 3));
+        EXPECT_EQ(xt::cast<std::size_t>(xt::view(py_a0, 2, all(), all())), xt::view(a0, 2, all(), all()));
+        EXPECT_EQ(xt::cast<std::size_t>(xt::view(py_a1, all(), 4, all())), xt::view(a1, all(), 4, all()));
+        EXPECT_EQ(xt::cast<std::size_t>(xt::view(py_a2, all(), all(), 3)), xt::view(a2, all(), all(), 3));
```

Comment:
I don't really mind, but just to know. Why are the casts needed here?

Reply:
Ditto

Reply:
OK!